### PR TITLE
fixed Issue 280 Terraform 0.13

### LINKED ISSUE #280: local storage configuration for profiles - importConfiguration
Is it possible to add this rest api attribute to TF:

importConfiguration
https://techlibrary.hpe.com/docs/enterprise/servers/oneview5.0/cicf-api/en/index.html#rest/server-profiles

This boolean value allows you to import disks of existing blades when creating profile.

@AsisBagga Thank you for your assistance in this and all the other issues!

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Enhancement Request for SP and SPT to add ImportConfiguration attribute

### Issues Resolved
#280 

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Bug fixes & Enhancements:
 - [#277] (https://github.com/HewlettPackard/terraform-provider-oneview/issues/277) SPT Import configuration issue
+- [#280] (https://github.com/HewlettPackard/terraform-provider-oneview/issues/280) local storage configuration for profiles - importConfiguration
```
Comment (VenkateshRavula) [inline CHANGELOG.md:10]: Add this bug under ```[v6.1.0-13]``` since that will be the next released version and this fix will be part of the next release.

**Review by VenkateshRavula — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AsisBagga) [reply CHANGELOG.md]: done

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -26,7 +26,10 @@ jobs:
             - name: Checkout code
               uses: actions/checkout@v2
             - name: Format  
-              run: if [ ! -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then exit 1; fi
+              run: if ! [ -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then
+                     echo  -e " $(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)";
```
Comment (nabhajit-ray) [inline .github/workflows/test.yml:30]: Please fix the gofmt check.

**Review by nabhajit-ray — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (AsisBagga) [reply .github/workflows/test.yml:30]: The above one works fine, I verified it from old travis and in 0.11TF workflow I tested it and it was printing out the unformatted files.

**Pull request merged**